### DISCUSSION TITLE
theme/foundations/sizes - largeSizes VS container - when to use container's sizes?

### DISCUSSION THREAD

**Discussion by hems** [chakra-ui/chakra-ui#2752] [Q&A]
I would like to know what's the difference between `largeSizes` and `container` and when they're used?

I noticed when giving "width" to `<Flex>` component it uses `largeSizes` but i don't know when chakra uses `container` sizes?

Here is a link to chakra's theme's source code: [foundations/sizes.ts#L3-L25](https://github.com/chakra-ui/chakra-ui/blob/develop/packages/theme/src/foundations/sizes.ts#L3-L25)

**Comment by with-heart** [comment]
`Container` should use `sizes.container` but there's currently a bug: #2730
